services/common/src/components/forms/RenderMultiSelect.tsx

```diff
@@ -35,8 +56,14 @@ export const RenderMultiSelect: FC<MultiSelectProps> = (props) => {
       {(value: IFormContext) => {
         const { isEditMode, isModal } = value;
         if (!isEditMode) {
-          const stringValue = "";
-          return <BaseViewInput value={stringValue} label={label} />;
+          const selectedOptions = data.filter((opt) => input.value.includes(opt.value));
```

Comment:
I was going to use the multi select at first so I made a view mode. I ended up not using it but it works so I committed it anyway. 🤷‍♀️